src/Plot/RobustPlot.purs

```diff
@@ -23,11 +23,11 @@ drawRobustPlot canvasSize numberOfPlots index bounds expression label = drawComm
   where
   f = evaluateWithX expression
 
-  points = plotPoints canvasSize bounds f
+  segmentEnclosures = plotPoints canvasSize bounds f
```

Comment:
In this context the renaming should be undone.

Reply:
`ctrl+f` has failed me. I will sort that out

Reply:
Sorry, I think I misunderstood the context.  Your subsequent further renaming is a better idea.